tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,199 @@
+import logging
+import hail as hl
+import hail.expr.aggregators as agg
+
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+from os.path import dirname
+
+from gnomad.utils.reference_genome import get_reference_genome
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("sex_check")
+logger.setLevel(logging.INFO) 
+def get_chr_cov(mt: hl.MatrixTable, build: str, call_rate_threshold: float=0.25, chr_name: str) -> hl.Table:
```

Comment:
```suggestion
logger.setLevel(logging.INFO) 


def get_chr_cov(mt: hl.MatrixTable, build: str, call_rate_threshold: float=0.25, chr_name: str) -> hl.Table:
```
adding more new lines to space out the code (this is a convention and makes it easier to read)